### ISSUE TITLE
[BUG] .gif animation in chat

### ISSUE THREAD

**Issue by keybreak** [oxen-io/session-desktop#2968]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-desktop/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

.gif file sent to Session chat doesn't animate on desktop

### Expected Behavior

.gif file sent to Session chat should be animated and auto-played on desktop, like it does on Android

### Desktop Version

v1.11.4

### Anything else?

Arch Linux, AUR

**Comment by KeeJef** [comment]
This one seems strange, could be a good first issue to investigate why this is occurring as GIFs should render as animated

**Comment by keybreak** [comment]
Yeah that's pretty weird, btw it animates in context of same gifs opened in lightbox, so it's only chat

**Comment by burtonemily** [comment]
@KeeJef Took a look at this issue and seems to be a more complex issue than just enabling autoPlay for GIF message types. We're currently generating a random src, so in context we can't tell what type of image the media is, without knowing, I can't set a condition for autoplay to be enabled (or to disabled the disabling of autoplay).
Probably going to take longer than just the day that I've assigned to it. 
(Should probably put this into the backlog for further investigation)

**Comment by KeeJef** [comment]
Tracked internally as SES-1487
